sign.proj

```diff
@@ -39,6 +39,15 @@
     </ItemGroup>
 
     <ItemGroup>
+      <FilesToSign Include="$(OutDir)$(ConfigurationGroup)/**/Microsoft.DotNet.PlatformAbstractions.dll">
```

Comment:
Sounds reasonable.  The `forPackaging` folder did stand out to me when I saw it in sign.proj because I didn't see it being used anywhere in the build.  I didn't want to do any cleanup in release/2.0.0 though, so I ignored it.  I'll change the `BaseOutputPath` as you suggested